### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by riensen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,108 @@
...
-1. Add your readme from your third party plugin to the folder you created
-1. Add your plugin to the root readme with a description and a link to your plugin specific readme
-1. Add your plugin's python package requirements to `requirements.txt`
-1. Add tests to get your plugin to 80% code coverage
+2. Make a folder for your plugin under `src/autogpt_plugins`. Name it a simple descriptive name such as `notion`, `twitter`, or `web_ui`.
+3. Take the files from your third-party plugin located at `src/auto_gpt_plugin_template` and add them into the folder you created
+4. Add your readme from your third-party plugin to the folder you created
+5. Add your plugin to the root readme with a description and a link to your plugin-specific readme
+6. Add your plugin's Python package requirements to `requirements.txt`
+7. Add tests to get your plugin to 80% code coverage
```
Comment (ntindle) [inline README.md:104]: Change these back to 1. for each so they are autonumbered by the markdown tool

File: README.md [hunk 2]
```diff
@@ -1,78 +1,108 @@
...
 ### First Party How-To
-1. Clone the pluigins repo
-1. Follow the structure of the other plugins, implementing the plugin interface as required
-1. Write your tests
-1. Make a PR back to this repo!
+
+1. Clone the plugins repo
+2. Follow the structure of the other plugins, implementing the plugin interface as required
+3. Write your tests
+4. Make a PR back to this repo!
```
Comment (ntindle) [inline README.md:87]: Changes these to 1. for all so they are auto incremented by the markdown tool

**Review by ntindle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (riensen) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (riensen) [reply README.md]: done

**Review by ntindle — APPROVED** [review]

**Pull request merged**